server/public/index.html

```diff
@@ -6,6 +6,11 @@
   <title>Final Project</title>
   <link rel="icon" href="/favicon.ico">
   <link rel="stylesheet" href="/styles.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
```

Comment:
I recommend putting your own styles _after_ external styles, just to be sure your own take priority.